src/swap/types.ts

```diff
@@ -267,6 +269,7 @@ export type SwapProviderReact = {
   experimental: {
     useAggregator: boolean; // Whether to use a DEX aggregator. (default: true)
   };
+  isSponsored?: boolean; // Whether the swap is sponsored. (default: false)
```

Comment:
nit: starting the comment description with `// An optional...`

Reply:
done